db/migrate/20150621061709_create_users.rb

```diff
@@ -0,0 +1,12 @@
+class CreateUsers < ActiveRecord::Migration
+  def change
```

Comment:
__[reek]__
* [TooManyStatements] has approx 6 statements
* [FeatureEnvy] refers to t more than self